src/app/tests/suites/certification/Test_TC_DRLK_2_1.yaml

```diff
@@ -1551,20 +1551,288 @@ tests:
       response:
           value: NumberOfCredentialsSupportedPerUserValue
 
-    - label: "Cleanup the created user"
-      PICS: DRLK.S.F00 && DRLK.S.F07
+    - label: "Step 36: TH sends ClearCredential Command to DUT"
+      PICS: DRLK.S.F00 && DRLK.S.F07 && DRLK.S.C26.Rsp
```

Comment:
Pre-existing, kind of, but this does not match the test plan.  Test plan has F08, not F07.

Reply:
will be fixed in a follow-up PR, tracking issue https://github.com/project-chip/connectedhomeip/issues/34710